backend/router/club.ts

```diff
@@ -21,20 +21,43 @@ router.get('/', async (req, res) => {
 //Adds a club with req.body
 router.post('/', async (req, res) => {
   const clubsCollection = await db.collection('clubs');
-  const clubsDoc = clubsCollection.doc();
+  const clubsDoc = clubsCollection.doc(req.body.id);
   console.log(req.body);
   const club: ClubType = req.body;
   await clubsDoc.set(club);
   res.send(club);
 });
 
+//Adds a club's social links (platform is other for custom social medias)
+router.post('/:id/socials/', async (req, res) => {
+  const clubID = req.body.id;
```

Comment:
This should use params and not req.body, as you are fetching the club id from the endpoint parameters (/:id) and not the body.

Reply:
this should be fixed now!